core/src/miner/miner.rs

```diff
@@ -232,13 +248,15 @@ impl Miner {
         }
     }
 
-    fn add_transactions_to_pool<C: AccountData + BlockChain + RegularKeyOwner>(
+    fn add_transactions_to_pool<C>(
```

Comment:
Do you need to change it?

Reply:
When I added a trait there, formatter automatically changed the style. Now I deleted that trait because it doesn't have to be there, but the style is not reverted. I'll revert it.